docker/simulation/carla_viz/carla_viz.Dockerfile

```
@@ -0,0 +1,75 @@
+FROM ubuntu:18.04 as repo
+RUN apt-get update -y && apt-get -y install git
+RUN git clone https://github.com/mjxu96/carlaviz.git && \
+    cd carlaviz && git checkout a8d6e66f4af91a7af5fe8169a03cc089c33ff62e
+WORKDIR /
+RUN git clone https://github.com/mjxu96/xviz.git && \
+    cd xviz && git checkout 6a45a56a7e62489d2200182a3379bb76fbf71141
```

Ideally, we wouldn't want git inside the container. You can separate git from the image by doing a multistage build
```suggestion
FROM ubuntu:18.04 as source
RUN apt-get update -y && apt-get -y install git
RUN git clone https://github.com/mjxu96/carlaviz.git && \
    cd carlaviz && git checkout a8d6e66f4af91a7af5fe8169a03cc089c33ff62e
WORKDIR /
RUN git clone https://github.com/mjxu96/xviz.git && \
    cd xviz && git checkout 6a45a56a7e62489d2200182a3379bb76fbf71141
   
FROM ubuntu:18.04 as repo
COPY --from=source carlaviz carlaviz
COPY --from=source xviz xviz
```

---

oh wait, it this separated out already?

---

oh yeah it is

---

idk, ill keep this in for an FYI